lib/view_component/action_textable.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module ActionTextable
+    if require("action_text")
```

Comment:
This doesn't look like the correct `if` predicate. This is going to unconditionally require `action_text`, which returns `true` if it is the first time it's being required and `false` otherwise. This would have the undesired effect of requiring `action_text` even in apps that don't use (and don't want to load) it.

Instead, I think we'd want to check for a constant that is only defined if the application has loaded `action_test/engine` - maybe `if defined?(ActionText::Engine)` but I haven't tested that myself. (You'd want to test in an app that has ActionText and in one that doesn't, and make sure it behaves correctly.)